docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  nginx-proxy:
+    image: nginx:1.17.6-alpine
+    ports:
+      - '127.0.0.1:9083:9083'
+    environment:
+      - ACCESS_CONTROL_ALLOW_ORIGIN=http://localhost:9082
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+      - ./nginx_envsubst.conf.template:/var/lib/hypothesis/nginx_envsubst.conf.template
+    command: bin/sh -c "envsubst '$${ACCESS_CONTROL_ALLOW_ORIGIN}' < /var/lib/hypothesis/nginx_envsubst.conf.template > /var/lib/hypothesis/nginx_envsubst.conf && exec nginx"
```

Comment:
Is this required or can we removed the command and rely on the `CMD` in the docker file? I tried it locally and it seems to work?

Reply:
This is referencing the official nginx image not the custom Dockerfile we created in this repo. (Tested via make services.) I guess if the official image has that exact command but I doubt that’s the case as that template file it’s replacing env vars in is AFAIK a custom name.

Reply:
Right because this is just used to run this locally, and this avoids having to install docker manually.